hub/core/index/index.py

```diff
@@ -267,8 +273,12 @@ def apply(self, array: np.ndarray):
 
         return array[index_values]
 
+    def is_trivial(self):
+        return (len(self.values) == 1) and self.values[0].is_trivial()
+
     def __str__(self):
-        return f"Index(" + str(self.values) + ")"
+        values = [entry.value for entry in self.values]
+        return f"Index({values})"
 
     def __repr__(self):
-        return self.__str__()
+        return f"Index(values={self.values})"
```

Comment:
is this different from `__str__` on purpose?

Reply:
Yes!

Reply:
Reason it that we want `__str__` to look pretty:
`Index([slice(1, 2, 1)])`
But `__repr__` should be the full expansion:
`Index(values=[IndexEntry(slice(1, 2, 1))])`